frontend/views/containers/contributions/IncomeDetails.vue

```diff
@@ -167,10 +163,15 @@ export default {
       this.$refs.modal.close()
     },
     async submit () {
-      if (this.$v.form.$invalid) {
-        alert(L('Invalid payment info'))
+      const debugThis = 1 + 1 // DELETE THIS BEFORE MERGE!!
+      if (debugThis || this.$v.form.$invalid) {
+        this.$set(this.form, 'submitFeedback', {
+          message: L('Please, review your information and try again.'),
+          severity: 'danger'
+        })
```

Comment:
Must this use `this.$set`? Is there an issue with `this.form.submitFeedback = `?

Reply:
Note: can ignore above comment if [this other approach](https://github.com/okTurtles/group-income-simple/pull/779/files#r348781980) works, since we won't need to have `submitFeedback` as a data variable on the parent